build/dockerfiles/rhel.Dockerfile

```diff
@@ -50,16 +50,8 @@ RUN ./check_mandatory_fields.sh devfiles
 
 RUN ./index.sh > /build/devfiles/index.json
 RUN ./list_referenced_images.sh devfiles > /build/devfiles/external_images.txt
-RUN chmod -R g+rwX /build/devfiles
-
-
-FROM registry.access.redhat.com/ubi8/nodejs-16-minimal:1-6 as dwtemplates
```

Comment:
this orphaned layer won't ever be updated with updateBaseImages because there's no associated RHEC URL from which to search for new tags